### PR TITLE
Pruned out old-style config model files for devicesim, testdevice-[12].x

### PR DESCRIPTION
**Pull request by tomikazi**

### CODE REVIEW

**Review by adibrastegarnia — APPROVED** [review]

File: config-models/stratum-1.x/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Deprecated
+This directory contains the old structure of the Stratum configuration models.
```
Comment (SeanCondon) [inline config-models/stratum-1.x/README.md:2]: adding this file means you will have to update the version number in `Chart.yaml` for this directory

**Review by SeanCondon — APPROVED** [review]
You will have to update the `onos-umbrella/Chart.yaml` too to remove these models, and increment its version number

File: onos-classic/values.yaml [hunk 2]
```diff
@@ -52,7 +52,7 @@ ports:
     port: 8181
 
 # logging:
-#   karafVersion: 4.2.9
+#   karafVersion: 4.2.14
```
Comment (SeanCondon) [inline onos-classic/values.yaml:55]: are there some merge problems going on hre - this is already in `master` branch

File: onos-umbrella/files/opa-rbac/devicesim-1.0.0.rego [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: 2022-present Open Networking Foundation <info@opennetworking.org>
```
Comment (SeanCondon) [inline onos-umbrella/files/opa-rbac/devicesim-1.0.0.rego:1]: and this is already on the `master` branch

**Comment by tomikazi** [comment]
Teo, thanks for fixing this. Can we merge this now?

**Pull request merged**